web/src/index.html

```diff
@@ -59,7 +59,7 @@
         var IG = function() {
```

Comment:
Here instead of IG function you can declare a object literal and its not necessary to declare it as `var` since its anyway global

e.g.
`
window.IG = {
api: 'your api',
cognitoUserPoolId: 'user pool id'
};
`